core/cloudflow-cli/src/main/scala/akka/cli/cloudflow/kubeclient/KubeClientFabric8.scala

```diff
@@ -209,18 +209,30 @@ class KubeClientFabric8(
       }
   }
 
-  def getOperatorProtocolVersion(): Try[String] = withClient { client =>
+  def getOperatorProtocolVersion(namespace: Option[String]): Try[String] = withClient { client =>
+    val secrets = namespace match {
+      case Some(ns) => client.secrets().inNamespace(ns)
+      case _        => client.secrets().inAnyNamespace()
+    }
+
     for {
-      crd <- Try {
-        client
-          .apiextensions()
-          .v1beta1()
-          .customResourceDefinitions()
-          .withName(App.ResourceName)
-          .get()
+      protocolVersionSecret <- Try {
+        secrets
+          .withLabel(App.CloudflowProtocolVersion)
+          .list()
+          .getItems()
+      }
+      protocolVersion <- {
+        protocolVersionSecret.size() match {
+          case 1 => Success(protocolVersionSecret.get(0))
+          case x if x > 1 =>
+            Failure(
+              CliException("Multiple Cloudflow operators detected in the cluster. This is not supported. Exiting"))
```

Comment:
It sort of is supported, but only when using `operatorNamespace`? (maybe specify that they must use the operatorNamespace flag instead?)